components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- The page html title (i.e. the <title> tag content) -->
+    <string name="error_connectionfailure_title">Unable to connect</string>
```

Comment:
@pocmo is `mozac` our a-c prefix?

Reply:
That's something we should document, I guess. :)

I tried to use `mozac` so that we do not collide with the consuming apps. But I also tried to use the module name so that we do not have collisions between components, e.g.:

```
mozac_browser_toolbar_menu_button -> String in browser-toolbar
mozac_feature_tabs_toolbar_tabs_button -> String in feature-tabs
```

But, yeah, for now just keep the names from Focus. :)